crates/algokit_transact_ffi/src/multisig.rs

```diff
@@ -0,0 +1,165 @@
+use crate::*;
+
+/// Representation of an Algorand multisignature signature.
+///
+/// This struct mirrors the Rust [`algokit_transact::MultisigSignature`] type, but uses FFI-safe types
+/// for cross-language interoperability.
+#[ffi_record]
+pub struct MultisigSignature {
+    /// The derived address for the multisignature group.
+    address: String,
```

Comment:
I'm not sure this should be a field, as changing anything else in the type invalidates this address.

Could we instead expose a function to get the address? The JS algosdk has `multisigAddress(multiSig: MultisigMetadata)`

Reply:
@PatrickDinh @joe-p